### DISCUSSION TITLE
FileInput ClearValue and Select Old File

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#2070] [Q&A]
FileInput
When clearvalue and select old file
it's not set value old file

https://codesandbox.io/s/pensive-cookies-8br3tq?file=/src/App.js

https://user-images.githubusercontent.com/12431734/184066400-9960bc48-00dc-4c8a-b8f8-4030ddef2a73.mov

**Comment by rtivital** [comment]
It seems like a bug, I'll see if I can fix that in one of the next patches

**Reply by goodlythink** [reply]
thanks

**Reply by ghost** [reply]
Is this Solved Yet.
